mmv1/templates/terraform/post_create/privateca_authority_enable.go.erb

```diff
@@ -1,4 +1,4 @@
-if d.Get("type").(string) != "SUBORDINATE" {
```

Comment:
By not enabling here, the resource will always resolve in a `STAGED` state?
It seems like this behavior would have been better captured as a `enable_on_create` field. However that would be a breaking change at this point.

Reply:
> By not enabling here, the resource will always resolve in a `STAGED` state?
Correct for root CA. For subordinated CA, it will be created in `PENDING_ACTIVATION`, which is the current behavior.

> It seems like this behavior would have been better captured as a `enable_on_create` field. However that would be a breaking change at this point.
Correct. Change in this PR would avoid a breaking change.